pkg/config.go

```diff
@@ -52,7 +62,58 @@ func ParseConfig(kommonsClient *kommons.Client, configFile string) ([]logs.Searc
 			}
 			backends = append(backends, backend)
 		}
+
+		if backend.ElasticSearch != nil {
+			cfg, err := getElasticConfig(backend.ElasticSearch)
+			if err != nil {
+				return nil, fmt.Errorf("error getting the elastic search config: %w", err)
+			}
+
+			client, err := v8.NewClient(*cfg)
+			if err != nil {
+				return nil, fmt.Errorf("error creating the elastic search client: %w", err)
+			}
+
+			pingResp, err := client.Ping()
```

Comment:
@moshloop Do we need to validate proper connection with the server before using it as a search backend?

Reply:
Yes, it makes sense